src/Colourista/Pure.hs

```diff
@@ -208,6 +209,10 @@ italic = fromString $ setSGRCode [SetItalicized True]
 {-# SPECIALIZE italic :: Text       #-}
 {-# SPECIALIZE italic :: ByteString #-}
 
+-- | Creates an indentation
+indent :: IsString str => Int -> str
+indent amount = fromString $ replicate amount ' '
```

Comment:
It is better to use `stimes` in here. It is more efficient.
Also, you would be able to get rid of `fromString` 🙂

Reply:
Can you please add `SPECIALIZE` pragmas as well?